File: java/blackjack/Hand.java
package blackjack;

import java.util.ArrayList;

/**
 * Created by 1mike12 on 12/16/2016.
 */
public class Hand {

    ArrayList<Card> cards = new ArrayList<>();

    Hand() {

    }

    void addCard(Card card) {
        cards.add(card);
    }

    int getHandScore() {
        int score = 0;

        for (Card card : cards) {
            score = score + card.getScore();
        }
        return score;
    }

    boolean isBusted() {
        return getHandScore() > 21;
    }
}

File: python2/blackjack/Hand.py
class Hand(object):
    def __init__(self):
        # list of cards
        self.cards = []

    def add_card(self, card):
        self.cards.append(card)

    def print_hand(self):
        print
        self.cards

    def get_score(self):
        total_score = 0
        for card in self.cards:
            total_score += card.score
        return total_score

    def is_busted(self):
        score = self.get_score()
        if score > 21:
            return True
        else:
            return False
